habitat-aspnet-sample/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=habitat-aspnet-sample
+pkg_origin=core
+pkg_version=0.1.9
+pkg_source=https://github.com/mwrock/${pkg_name}/archive/v${pkg_version}.tar.gz
+pkg_shasum=50ffcbe1c0b49acb84365c2157445047b360345c7361e67827093f962852de4d
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_description="A sample asp.net core app for linux"
+pkg_bin_dirs=(bin)
+pkg_expose=(8090)
+
+pkg_deps=(
+  core/dotnet-core
+  core/gcc
+  core/glibc
+  core/gcc-libs
+  core/icu/52.1
+  core/util-linux
+  core/krb5
+  core/libunwind
+  core/lttng-ust
+  core/openssl
+)
+
+pkg_build_deps=(
+  core/node
+  core/patchelf
+)
+
+do_build() {
+  npm install -g gulp
```

Comment:
Its also possible that there might be some other way to avoid `-g` like manually adding them to the path. I have not worked much with node in the past and the `-g` just worked but I'll poke more at this.